en/getting-started.md

```diff
@@ -1,10 +1,106 @@
 # Getting started
 
+## Main Window of JabRef
+
+Upon the first start of JabRef the main user interface is showing up the main elements are:
+
+- Menu bar
+- Icon bar (shortcuts for most frequently used features)
+- Side bar (for groups and web search)
+
+![Screenshot of main window](.gitbook/assets/getting-started-main-screen.png)
+
+## Creation of a new library
+
+A "library" is the main file that saves all the information about your collection of references. The storage format of the file is text-based in the Bibtex standard.
```

Comment:
I would move this information about bibtex to https://docs.jabref.org/cite/bibtex-and-biblatex. It's not utterly important for a new user how JabRef saves his information (except when he starts to cite papers in tex).

Reply:
As this is for me still one of the essential aspects of JabRef I would keep it ;-)